Xamarin.Forms.Platform.Android/Renderers/ListViewAdapter.cs

```diff
@@ -403,20 +403,9 @@ List<Cell> GetCellsFromPosition(int position, int take)
 
 				if (global == position || cells.Count > 0)
 				{
-					if (_listView.CachingStrategy == ListViewCachingStrategy.RecycleElement)
-					{
-						var groupContent = _listView.TemplatedItems.GroupHeaderTemplate?.CreateContent(group.ItemsSource, _listView) as Cell;
-						if (groupContent != null)
-						{
-							groupContent.Parent = _listView;
-							groupContent.BindingContext = group.ItemsSource;
-							cells.Add(groupContent);
-						}
-					}
-					else
-					{
-						cells.Add(group.HeaderContent);
-					}
+					//Always create a new cell if we are using the RecycleElement strategy
```

Comment:
This is the most strange comment I've ever seen...

What's the purpose of using `Recycle` if we recreate cells ?